src/tree/AttachedAccountsTreeItem.ts

```diff
@@ -246,18 +249,23 @@ export class AttachedAccountsTreeItem implements IAzureParentTreeItem {
     private async persistIds() {
         const value: IPersistedAccount[] = this._attachedAccounts.map((node: IAzureTreeItem) => {
             let experience: Experience;
+            let isEmulator: boolean;
             if (node instanceof MongoAccountTreeItem) {
                 experience = Experience.MongoDB;
+                isEmulator = node.isEmulator;
             } else if (node instanceof GraphAccountTreeItem) {
                 experience = Experience.Graph;
+                isEmulator = node.isEmulator;
             } else if (node instanceof TableAccountTreeItem) {
                 experience = Experience.Table;
+                isEmulator = node.isEmulator;
             } else if (node instanceof DocDBAccountTreeItem) {
                 experience = Experience.DocumentDB;
+                isEmulator = node.isEmulator;
```

Comment:
Should I place all the `isEmulator = node.isEmulator` lines within one large check for all AccountTreeItems?